addons/web/static/src/js/views/control_panel/control_panel_model.js

```diff
@@ -1105,6 +1105,17 @@ var ControlPanelModel = mvc.Model.extend({
                             asc: asc,
                         };
                     });
+
+                    var stringified_context = JSON.stringify(context);
+                    // `context` is a stringified Python dict.
+                    // This replaces instances of booleans `true` and `false` by `True` and `False`.
+                    stringified_context = stringified_context.replaceAll(
+                        /(true|false)(?=(?:\\.|[^"'\\])*(["'](?:\\.|[^'"\\])*["'](?:\\.|[^'"\\])*)*$)/g,
```

Comment:
Shouldn't that work: `\b(true|false)\b`?

Reply:
It'd match words in strings.
`{"test": "test is true or false and shouldn't match"}`